public/template/layout.jade

```diff
@@ -64,90 +65,92 @@ html(lang="en")
         p
           a(href="http://pump.io/") pump.io
           |  available under the Apache License 2.0.
-        p
-          a(href="/javascript/about.html", data-jslicense="1", data-bypass) JavaScript license information
-
-    if config.debugClient
-      script(src="/javascript/libs/jquery-1.10.2.js")
-      script(src="/javascript/libs/jquery.easydate-0.2.4.js")
-      script(src="/javascript/libs/bootstrap.js")
-      script(src="/shared/lodash.js")
-      script(src="/javascript/libs/backbone.js")
-      script(src="/javascript/libs/oauth.js")
-      script(src="/javascript/libs/sha1.js")
-      script(src="/javascript/libs/spin.js")
-      script(src="/javascript/libs/jquery-spin.js")
-      script(src="/javascript/libs/jquery.fineuploader.js")
-      script(src="/javascript/libs/wysihtml5-0.3.0.js")
-      script(src="/javascript/libs/bootstrap-wysihtml5.js")
-      script(src="/javascript/libs/select2.js")
-      script(src="/javascript/libs/bootstrap-lightbox.js")
-      if config.sockjs
-        script(src="/javascript/libs/sockjs.js")
-    else
-      if config.noCDN
-        script(src="/javascript/libs/jquery-1.10.2.min.js")
-        script(src="/javascript/libs/bootstrap.min.js")
-        script(src="/shared/lodash-min.js")
-        script(src="/javascript/libs/backbone-min.js")
-        script(src="/javascript/libs/spin.min.js")
-        script(src="/javascript/libs/jquery.fineuploader.min.js")
-        script(src="/javascript/libs/wysihtml5-0.3.0.min.js")
-        script(src="/javascript/libs/select2.min.js")
+        if !config.noweb
+          p
+            a(href="/javascript/about.html", data-jslicense="1", data-bypass) JavaScript license information
+          
+    if !config.noweb
+      if config.debugClient
+        script(src="/javascript/libs/jquery-1.10.2.js")
+        script(src="/javascript/libs/jquery.easydate-0.2.4.js")
+        script(src="/javascript/libs/bootstrap.js")
+        script(src="/shared/lodash.js")
+        script(src="/javascript/libs/backbone.js")
+        script(src="/javascript/libs/oauth.js")
+        script(src="/javascript/libs/sha1.js")
+        script(src="/javascript/libs/spin.js")
+        script(src="/javascript/libs/jquery-spin.js")
+        script(src="/javascript/libs/jquery.fineuploader.js")
+        script(src="/javascript/libs/wysihtml5-0.3.0.js")
+        script(src="/javascript/libs/bootstrap-wysihtml5.js")
+        script(src="/javascript/libs/select2.js")
+        script(src="/javascript/libs/bootstrap-lightbox.js")
         if config.sockjs
-          script(src="/javascript/libs/sockjs.min.js")
```

Comment:
This seems to be an accidental deletion? And this `if` block should be indented too I think

Reply:
No, this will not delete, is for indentation change, see: https://github.com/pump-io/pump.io/pull/1446/files#diff-4f8f8fb714a7dfc046972c1880ae5170R101

Reply:
Ahhh, sorry - I thought you'd deleted the `.min` part but it's actually GitHub just not rendering a good diff. My bad!